server/routes/allCompletedReports/allCompletedReports.ts

```diff
@@ -31,7 +31,7 @@ export default class AllCompletedReportsRoutes {
   ): Promise<void> =>
     res.render(`pages/allCompletedReports`, {
       allCompletedReports: results,
-      filter: filter == null ? [{ status: 'SCHEDULED' }] : filter,
+      filter: filter === null ? [{ status: `allStatuses` }] : filter,
```

Comment:
You can just pass `filter` in here. No need for the ternary. This is because filter should never be equal to `null` - if you look at the functions involved this this line `const uiFilter = fillInDefaults(uiFilterFromRequest(req))`, it pulls the dates and statuses from the request, and if there's nothing there, default dates are passed in and the array of all the statuses.
You can test this by removing the ternary here and going to `/all-completed-reports` - all the statuses are checked even though there's no query parameters